### ISSUE TITLE
Short Video of the Keyboard and Thumb Sticks in Action

### ISSUE THREAD

**Issue by rwxOsu** [dschil138/Fulcrum#8]
Hello, 
would it be possible for you to upload a short video demonstrating the keyboard with the thumb clusters in operation? It's difficult to determine from the graphics alone whether the thumb keys would be the right fit for me and how the ergonomics work. It's a great idea and a cool design though! Best regards

**Comment by dschil138** [comment]
Yeah, a few people have requested this. I've tried to make one but it's surprisingly difficult to get an angle where you can actually see how the thumbs interact with the joysticks without it being blocked by the other parts of your hand while typing. Harder especially bc I don't have a tripod or a smartphone mount. This is at the top of my to do list for the project tho, I am hoping to get a video done this week. I'll post it here when I do!
